Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,26 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail tempTrail;
```

Comment:
mind the names; not a good practice

Reply:
@gusgolko naming sounds reasonable atm, but what do you suggest?